### PR TITLE
LIBCLOUD-288 Gandi on libcloud http

### PR DESCRIPTION
**Pull request by Jc2k**
This makes the Gandi xmlrpclib driver work much more like a normal driver, and it makes full use of the libcloud http features.

### CODE REVIEW

File: libcloud/common/gandi.py [hunk 1]
```diff
@@ -104,6 +67,27 @@ def parse_error(self, code, message):
         pass
 
 
+class GandiConnection(ConnectionKey):
+    """
+    Connection class for the Gandi driver
+    """
+
+    responseCls = GandiResponse
+    host = "rpc.gandi.net"
```
Comment (Kami) [inline libcloud/common/gandi.py]: Minor style thing - please use single quotes around the strings everywhere besides the docstrings :)

**Comment by Kami** [comment]
Just curious, do the tests pass with Python 3.2?

I know there were some changes to `xmlrpclib` in 3.2 which are usually causing troubles.

File: libcloud/common/gandi.py (hunk 1 above)
Comment (Jc2k) [inline libcloud/common/gandi.py]: Gah, I will get used to this. I will get used to this. I will get use to this.

It's for my own good, everything other bit of code I write must randomly hop between the two styles :-)

Comment (Kami) [inline libcloud/common/gandi.py]: heh :)

**Comment by Jc2k** [comment]
Good call.

The tests do seem to pass on 3.2, as does my out-of-tree script that goes and pokes around uncomfortably close to my production DNS :D

But i'll pay extra attention for 3.2 related issues.

**Comment by Kami** [comment]
Looks good to me then. Feel free to attach patch to the JIRA ticket when you are done with addressing all the comments.

**Comment by Jc2k** [comment]
Patch attached to JIRA, closing PR.

**Pull request closed without merging**